packages/shared/src/components/modals/BootPopups.tsx

```diff
@@ -30,15 +30,14 @@ export const BootPopups = (): null => {
   const [bootPopups, setBootPopups] = useState(() => new Map());
   const { getMarketingCta } = useBoot();
   const marketingCta = getMarketingCta(MarketingCtaVariant.Popover);
-  const { shouldShowPopup: shouldShowStreaksPopup } = useStreakExperiment();
-  const { streak, isEnabled } = useReadingStreak();
+  const { streak, shouldShowPopup: shouldShowStreaksPopup } =
+    useReadingStreak();
   const { loadedSettings, optOutWeeklyGoal } = useSettingsContext();
```

Comment:
I think this PR actually removes `optOutWeeklyGoal` from the settings context. Should this have been removed or I just do not understand things in the morning? 😊

Reply:
Yes true true, this can be removed

Reply:
Hmm, based on Lee's comment looks like we might want to keep it. Will check with product about this https://github.com/dailydotdev/apps/pull/3091#discussion_r1599502582